doc/01DMEDesign.md

```diff
@@ -0,0 +1,26 @@
+#DME-Core
+
+The DME-Core is a RISC OS App that contains both logic and resources to achieve the requirement goals.
+
+Directory tree:
+```
+!DMECore
+├── Resources
+│   └── UK
+└── Store
+    ├── Controls
+    ├── Gadgets
+    └── Themes
+```
+The Resources directory is the standard RISC OS repository for territory and messages as well as app templates (if neeved)
```

Comment:
```suggestion
The Resources directory is the standard RISC OS repository for territory and messages as well as app templates (if needed)
```
Also not sure repository is a good word, since we use that for the git repository as a whole.